src/components/blog/post/body.module.css

```diff
@@ -0,0 +1,358 @@
+  margin-left: auto;
+}
+
+/* Anchors */
+
+.root a,
+.root a:hover,
+.root a:link,
+.root a:visited {
+  color: #2421ab;
+
+  font-weight: 700;
+  text-decoration: none;
+}
+
+.root a:hover {
+  opacity: 0.5;
+
+  transition: opacity 0.2s;
+}
+
+/* Code samples */
+
+.root code {
+  font-family: source-code-pro, monospace;
+  font-weight: 400;
+}
+
+/* Headings */
+
+.root h2,
+.root h3,
+.root h4,
+.root h5,
+.root h6 {
+  font-family: colfax-web, sans-serif;
+  font-weight: 500;
+}
+
+.root h2 {
+  margin-top: 84px;
+  margin-bottom: 20px;
+
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.root h3 {
+  margin-top: 40px;
+  margin-bottom: 20px;
+
+  font-size: 20px;
+  line-height: 28px;
+}
+
+/* Paragraph */
+
+.root > p:first-child::first-letter {
+  float: left;
+  padding-right: 4px;
+  margin-top: 0;
+
+  font-family: "Acta Display", serif;
+  font-size: 56px;
+  line-height: 56px;
+}
+
+.root p:not(:first-child) {
+  margin-top: 20px;
+}
+
+.root p:not(:last-child) {
+  margin-bottom: 28px;
+}
+
+/* Images (in paragraphs) */
+
+.root p > img {
+  position: relative;
+  left: 50%;
+
+  display: block;
+  width: auto;
+  max-width: calc(100% + 40px);
+  margin: 40px 0;
+
+  transform: translateX(-50%);
+}
+
+/* Inline code */
+
+.root p code,
+.root li code {
+  padding: 0 4px;
+
+  background-color: rgba(64, 63, 76, .1);
+
+  font-size: smaller;
+}
+
+/* Quote blocks */
+
+.root blockquote {
+  position: relative;
+
+  padding: 20px 20px 0;
+  margin: 56px auto 40px 0;
+
+  font-family: 'Acta Headline', serif;
+  font-size: 20px;
+  font-weight: 800;
+  line-height: 28px;
+}
+
+.root blockquote::before {
+  content: '“';
+
+  position: absolute;
+  top: -26px;
+  left: 95px;
+
+  display: inline-block;
+
+  font-size: 56px;
+  font-weight: normal;
+  line-height: 56px;
+}
+
+.root blockquote p {
+  padding: 0;
+}
+
+.root blockquote p:not(:last-child) {
+  margin-bottom: 20px;
+}
+
+.root blockquote p:last-child:not(:first-child) {
+  width: calc(100% - 72px);
+  max-width: 506px;
+  margin-right: 0;
+  margin-left: auto;
+
+  font-family: colfax-web, sans-serif;
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 20px;
+}
+
+/* Code blocks */
+
+.root pre {
+  padding: 20px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+
+  background-color: rgba(64, 63, 76, .1);
+}
+
+.root pre > code {
+  display: block;
+  max-width: 578px;
+  padding: 0;
+  margin: 0 auto;
+
+  background-color: transparent;
+  border-radius: 0;
+
+  white-space: break-spaces;
+}
+
+/* Ordered and unordered lists */
+
+.root ol,
+.root ul {
+  padding-left: 40px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+/* Basic inline modifiers */
+
+.root em {
+  font-style: italic;
+}
+
+.root strong {
+  font-weight: bold;
+}
+
+/* Horizontal rule */
+
+.root hr {
+  width: 25vw;
+  max-width: 174px;
+  margin: 28px auto;
+
+  border-top: none;
+  border-bottom: currentColor solid 1px;
+}
+
+@media (min-width: 619px) {
+  /* Images (in paragraphs)
```

Comment:
:rotating_light:

Reply:
So, no comments is better? I'm honestly looking for any help making this file not look like a mess.

Reply:
No :joy: the formatting was weird, doesn't prettier fix that?

Reply:
On my setup, prettier does not touch what is not meant to be touched. And in our case only stylelint touches CSS files :stuck_out_tongue: 

Anyhow this is going away now :grin: